helm/management-cluster-admission/templates/kustomize-out.yaml

```diff
@@ -258,9 +258,9 @@ data:
     apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
     kind: ControllerManagerConfig
     health:
-      healthProbeBindAddress: :8081
+      healthProbeBindAddress: '{{ .Values.healthProbeBindAddress }}'
     metrics:
-      bindAddress: 127.0.0.1:8080
+      bindAddress: '127.0.0.1:{{ .Values.metricsBindAddress }}'
```

Comment:
Why doest that need to be configurable? If it really needs to then we'll have to change https://github.com/giantswarm/management-cluster-admission/tree/main/config/helm with some patch. Try running `make manifests` and you'll see everything here will be reverted.

Reply:
I thought it'd be good to make the change since they're configurable here: https://github.com/giantswarm/management-cluster-admission/blob/main/config/helm/manager_config.yaml
I can skip it though. 

I'll add other changes I made to `helm/` to `config/` to ensure it's generated properly as well.

Reply:
Okay sounds good. I didn't even realise this configuration exists. I'm not sure if this is read BTW. Possibly I removed configuration part without realising. But if you change anything in the chart that has to happen in `config/helm` otherwise it'll get revereted.